Multimedia player: Disable top border radii on details/summary

EricDunsworth opened the pull request:
When WET's media player uses an embedded transcript via expandable/collapsible content, it's intended to be situated directly below the media player's controls.

Previously, the expandable/collapsible content looked slightly-disconnected from the player due to its rounded top border. This change addresses it by disabling the top border radii.

**Screenshots...**

**Before:**
![multimedia-transcript-border-before](https://user-images.githubusercontent.com/1907279/109570482-159b4300-7ab8-11eb-9e82-d36add34abed.png)

**After:**
![multimedia-transcript-border-after](https://user-images.githubusercontent.com/1907279/109570486-16cc7000-7ab8-11eb-9237-8cc2ae9f8b0e.png)

EricDunsworth commented:
Btw the benefits of this change might look almost imperceptible at first glance, but it's a bit more obvious in GCWeb (due to its bigger font size making summaries taller) and will become much more obvious if details/summary borders get darkened in the future to be more on par with what was done with panels in #8955.

duboisp commented on src/plugins/multimedia/_base.scss:
```diff
@@ -74,6 +74,11 @@
 	vertical-align: middle;
 }
 
+%multmedia-details-summary-border-top-radius-0 {
+	border-top-left-radius: 0;
+	border-top-right-radius: 0;
+}
+
```
This became a mixin very very specify, only for 1 use case and don't seems to be re-usable in other context.

Please just repeat those 2 line in your CSS code at line 191 and line 196.

EricDunsworth replied:
The placeholder selector is being extended twice. Are you sure you want me to eliminate it?

duboisp reviewed (changes requested):
Other than removing the mixin,

 it seems good to go after testing.

duboisp commented on src/plugins/multimedia/_base.scss:
```diff
@@ -182,6 +187,15 @@
 		text-align: right;
 	}
 
+	summary {
```
Suggested changes:

```
summary, details[open] { }
```

EricDunsworth replied:
That's not in line with how WET has traditionally dealt with this kind of situation. Selector rules are usually placed on their own lines and separated as much as possible. That's why I had resorted to ``&[open]`` and a placeholder selector instead of ``details[open]``).

In any case, I'll go with the way you're suggesting to simplify the code and since I don't feel super strongly about it.

EricDunsworth committed:
Multimedia player: Disable top border radii on details/summary

When WET's media player uses an embedded transcript via expandable/collapsible content, it's intended to be situated directly below the media player's controls.

Previously, the expandable/collapsible content looked slightly-disconnected from the player due to its rounded top border. This change addresses it by disabling the top border radii.

GormFrank commented:
Approved as long as @duboisp approves.
